frame/staking/src/lib.rs

```diff
@@ -1484,18 +1533,31 @@ decl_module! {
 		///
 		/// # <weight>
 		/// - The transaction's complexity is proportional to the size of `targets`,
-		/// which is capped at CompactAssignments::LIMIT.
+		/// which is capped at CompactAssignments::LIMIT (MAX_NOMINATIONS).
 		/// - Both the reads and writes follow a similar pattern.
+		/// ---------
+		/// Base Weight: 22.34 + .36 * n µs
```

Comment:
nit: reader needs to infer meaning of `n`

Reply:
yeah I had the same situation. Not a big deal for now, but we have a dilemma: name them the same as we would in the benchmarks (`n`) or the real param name (`targets`). I usually chose the latter. 

all in all and long term I think it is very important to have a cohesive standard of how to write this `<weight>` document, down to the very small detail. It will make it easier to read and write + we can easily then write tools that scrape the `<weight>` part and do all sorts of cool stuff on top of it.

Reply:
I agree. The way I usually do it is that I have a complexity written in the weight docs (e.g. `O(N)`) and then define the parameters (e.g. "with `N` being the number of targets").